src/datepicker/types.js

```diff
@@ -201,7 +210,8 @@ export type DatepickerPropsT = CalendarPropsT & {
 };
 
 export type SharedStylePropsT = {
-  $date: Date,
+  // eslint-disable-next-line flowtype/no-weak-types
+  $date: any,
```

Comment:
I tried to swap this over to `T` but discovered that all style properties were getting `automatically` typed as `any` anyway, so I just made this any to make it consistent.